packages/atomic/src/components/common/interface/analytics-config.ts

```diff
@@ -55,17 +55,18 @@ export function getNextAnalyticsConfig(
   searchEngineConfig: EngineConfiguration,
   enabled: boolean
 ): AnalyticsConfiguration {
-  const defaultConfiguration: AnalyticsConfiguration = {
+  const configuration: AnalyticsConfiguration = {
     enabled,
     documentLocation: document.location.href,
     ...(document.referrer && {originLevel3: document.referrer}),
   };
 
-  if (searchEngineConfig.analytics) {
-    return {
-      ...defaultConfiguration,
-      ...searchEngineConfig.analytics,
-    };
-  }
-  return defaultConfiguration;
+  const analyticsConfiguration = searchEngineConfig.analytics ?? {};
```

Comment:
Also isnt analyticsConfiguration useless here ? It is assigned and then unused

Reply:
resolved w/ fred suggestion